sw/device/lib/crypto/BUILD

```diff
@@ -34,6 +34,15 @@ cc_import(
     ],
 )
 
+cc_import(
```

Comment:
nit: I think a comment here might be helpful to clarify the contexts in which this target should (or should not) be used. Is it only for tests within the OT repo of the freestanding/exported lib?

Reply:
Fixed: added a comment.

This target is used to test that the freestanding version of the library & headers work correctly on their own, whereas the `:crypto` target can be used to test cryptolib within the codebase.